authors/apps/authentication/tests/test_models.py

```diff
@@ -1,5 +1,8 @@
 from django.test import TestCase
 from ..models import User, UserManager
+import jwt
+from django.conf import settings
+from datetime import datetime, timedelta
 
```

Comment:
@lytes20 
- I would recommend importing **python modules**, **first then Django**, *then your own**

Reply:
cool, will start doing this moving forward